src/lib/app.js

```diff
@@ -50,7 +50,6 @@ class App {
     koaApp.use(compress())
 
     const websocketRouter = this._makeWebsocketRouter()
-    koaApp.ws.use(logger)
```

Comment:
Fixed. The issue is fixed in the new version of koa-riverpig, so I can leave the logging in.